server/certs.go

```diff
@@ -82,8 +82,9 @@ func generateMediaTLSCert() error {
 		return nil
 	}
 
-	notBefore := timesource.GetCurrentTime()
-	notAfter := notBefore.Add(365 * 24 * time.Hour)
+	now := timesource.GetCurrentTime()
```

Comment:
If the time range has expanded significantly, we may no longer need an accurate time. In this case, using the built-in time.Now() function might be preferable because the time source may require network access.

Reply:
fixed, thanks